test/common/upstream/subset_lb_test.cc

```diff
@@ -1686,10 +1686,10 @@ TEST_F(SubsetLoadBalancerTest, DescribeMetadata) {
   num_value.set_number_value(100);
 
   auto tester = SubsetLoadBalancerDescribeMetadataTester(lb_);
-  tester.test("version=\"abc\"", {{"version", str_value}});
+  tester.test("version=abc", {{"version", str_value}});
```

Comment:
Do we have some better tests for this? One thing to note is that JSON is a proper subset of YAML, so any JSON tests for logging would pass today. Probably want some golden config fragment comparison.